youtube_dl/extractor/minoto.py

```diff
@@ -26,7 +26,7 @@ def _real_extract(self, url):
                 continue
             container = fmt.get('container')
             if container == 'hls':
-                formats.extend(fmt_url, video_id, 'mp4', m3u8_id='hls', fatal=False)
+                formats.extend([fmt_url, video_id, 'mp4'], m3u8_id='hls', fatal=False)
```

Comment:
This is an actual bug; thanks for the hint:
```suggestion
                formats.extend(self._extract_m3u8_formats(fmt_url, video_id, 'mp4', m3u8_id='hls', fatal=False))
```
```